### DISCUSSION TITLE
Destinations

### DISCUSSION THREAD

**Discussion by matatk** [w3c/personalization-semantics#250] [Ideas]
We had a [range of destinations specified in the Content Module](https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/content/index.html#values-0). These are being used as the basis for the information architecture proposal. We should go back to COGA for an update/any questions on these. Let's use this thread to collect thoughts on them beforehand.

**Comment by abhina-kumar** [comment]
> We had a [range of destinations specified in the Content Module](https://raw.githack.com/w3c/adapt/CR-content-2022-07-26/content/index.html#values-0). These are being used as the basis for the information architecture proposal. We should go back to COGA for an update/any questions on these. Let's use this thread to collect thoughts on them beforehand.

@matatk : Thanks for the link. My concern is some of these are actually action rather than a page like `signout`, `post` etc. How will be the behaviour for these cases?

**Reply by matatk** [reply]
You're right; some I feel are more 'actions' than destinations - this is something we should follow up with [COGA](https://www.w3.org/WAI/GL/task-forces/coga/) as part of developing these specs.

**Reply by abhina-kumar** [reply]
@matatk: Thanks! Can we capture the follow-ups (whenever they happen) also in this thread?

**Comment by AutoSponge** [comment]
We need to include well-known report urls. 

For example, the `/accessibility-report` or `/accessibility/report` could accept someone's automatic or manual feedback about a barrier (see the separate explainer for that spec). More commonly, there are proposals for posting broken links, errors, etc. That can be completed by the browser or AT but the URIs need to be detectable/predictable. 

I think there was an idea to include these in page metadata or server headers, but a predictable URI and a quick "test" to make sure it's not just a naming collision or malicious page would be good. The test part is going to be a list of MUST/MUST NOT statements for these pages. IMO, landing on these pages without the passing test should warn the user that the page is untrusted (minimum) or block it.

**Reply by matatk** [reply]
I agree that mechanisms for accessibility issue reporting should be provided, and that doing it using a well-known method is good. Also, can appreciate that a well-known URL leading to a human-readable form might be good.

There is already an established [Reporting API](https://www.w3.org/TR/reporting-1/) to which we were discussing (as you remembered) adding accessibility-related endpoints. I think it's worth considering whether we should continue down that path, or perhaps the well-known URLs one (or both - though perhaps that could be confusing).

Perhaps worth looking at both the UX and DX implications, and potential privacy issues, of each path - and how well supported the Reporting infrastructure is (both in browsers and in the wild).

**Reply by abhina-kumar** [reply]
@matatk : Is URL of `Reporting API` fixed? 

From the Reporting API spec, I see;
`Each [report](https://www.w3.org/TR/reporting-1/#report) has a url, which is typically the address of the Document or Worker from which the report was generated.`

If the URL is not fixed, then it makes sense to make URL known via well-known URL

**Comment by Duy-De** [comment]
Ok

Vào 23:13, T.4, 13 Th12, 2023 Paul Grenier ***@***.***> đã
viết:

**Comment by Duy-De** [comment]
[image: Google My Maps]
Bản đồ không có tiêu đề
<https://www.google.com/maps/d/edit?mid=12nW6YhD9ANhKOBFwUKzLsJqIdX0EcMM>

[image: Google My Maps]
Bản đồ không có tiêu đề
<https://www.google.com/maps/d/edit?mid=12nW6YhD9ANhKOBFwUKzLsJqIdX0EcMM>
[image: Biểu mẫu Google]
Mẫu không có tiêu đề
<https://docs.google.com/forms/d/1csWSN3RHNq0eHlfU2UmBpgPj-Aaiq7ROCjyjJlGMLpg/edit>

---------- Thư đã chuyển tiếp ---------
Từ: Mail Delivery Subsystem ***@***.***
***@***.***>*>
Ngày: Th 3, 27 thg 8, 2024 lúc 12:08
Tiêu đề: Delivery Status Notification (Failure)
Đến: ***@***.***>


[image: Biểu tượng lỗi]
Không tìm thấy địa chỉ
Thư của bạn không được gửi đến ***@***.**** vì không thể tìm thấy địa
chỉ hoặc địa chỉ không thể nhận thư.
Phản hồi từ máy chủ từ xa là:

550 #5.1.0 Address rejected.



---------- Forwarded message ----------
From: "Đệ Phạm" ***@***.***>
To: ***@***.***
Cc:
Bcc:
Date: Tue, 27 Aug 2024 12:08:30 +0700
Subject: Quan tâm

**Comment by Duy-De** [comment]
***@***.***
